mona_sdk/mona_single_message.py

```diff
@@ -29,18 +32,32 @@ class MonaSingleMessage:
         message=<the relevant monitoring information>,
         contextClass="MY_CONTEXT_CLASS_NAME",
         contextId=<the context instance unique id>,
-        exportTimestamp=<the message export timestamp>,
+        sendTimestamp=<the message export timestamp>,
         action=<the wanted action>
     )
     """
 
+    def __post_init__(self):
+        if self.exportTimestamp is not None and self.sendTimestamp is None:
+
+            if ENABLE_DEPRECATION_WARNINGS_ON_EXPORT_TIMESTAMP:
+                warnings.warn(
+                    "'exportTimestamp' is deprecated. Use 'sendTimestamp' instead.",
+                    DeprecationWarning,
+                    stacklevel=2,
+                )
+
+            self.sendTimestamp = self.__dict__["exportTimestamp"]
```

Comment:
Maybe also remove "exportTimestamp" to make sure we already aren't really using it anymore